src/renderers/WebGLRenderer.js

```diff
@@ -1726,6 +1726,7 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 		// custom render plugins (post pass)
 
+		this.resetGLState();
```

Comment:
Yes, `initAttributes()` also works. I wasn't sure which method is preferred. I chose the stronger `resetGLState()` because all the plugins `render()` method end with `resetGLState()` themselves.